### ISSUE TITLE
@Blocking hides exception

### ISSUE THREAD

**Issue by zeljkot** [quarkusio/quarkus#10253]
**Describe the bug**

If message handler is annotated with `@Blocking` and it throws exception, the exception silently breaks the messaging. If `@Blocking` is removed, the exception is shown.

**Expected behavior**
Exception stack trace should be shown.

**Actual behavior**
Nothing is shown.

**To Reproduce**

```java
    @Incoming("kafka-something")
    @Blocking
    public void onMessage(byte[] message) {

        LOG.warn("Throwing it");
        throw new RuntimeException("Fakeception");
    }
```

Try commenting out `@Blocking`.

**Configuration**
```properties
# Add your application.properties here, if applicable.

```

**Screenshots**
(If applicable, add screenshots to help explain your problem.)

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Microsoft Windows [Version 10.0.19041.329]
 - Output of `java -version`: 
openjdk 11.0.6 2020-01-14
OpenJDK Runtime Environment GraalVM CE 20.0.0 (build 11.0.6+9-jvmci-20.0-b02)
OpenJDK 64-Bit Server VM GraalVM CE 20.0.0 (build 11.0.6+9-jvmci-20.0-b02, mixed mode, sharing)
 - GraalVM version (if different from Java): 
 - Quarkus version or git rev: 1.5.2.Final
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)

**Additional context**
(Add any other context about the problem here.)

@cescoffier

**Comment by cescoffier** [comment]
CC @kenfinnigan

**Comment by kenfinnigan** [comment]
I've created an issue in SmallRye Reactive Messaging and have a possible fix.

Working on ways to ensure it's properly validated

**Comment by cescoffier** [comment]
The upstream project (SmallRye Reactive Messaging) contains a fix.

**Comment by cescoffier** [comment]
Fixed by #10779
